assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -208,18 +209,20 @@ export default function WidgetAreaRenderer( { slug, contextID } ) {
 			key={ `${ widget.slug }-wrapper` }
 			gridColumnWidth={ gridColumnWidths[ i ] }
 		>
-			<WidgetRenderer
-				OverrideComponent={
-					overrideComponents[ i ]
-						? () => {
-								const { Component, metadata } =
-									overrideComponents[ i ];
-								return <Component { ...metadata } />;
-						  }
-						: undefined
-				}
-				slug={ widget.slug }
-			/>
+			<ErrorBoundary slug={ slug } Widget={ widget }>
```

Comment:
As discussed [here](https://github.com/google/site-kit-wp/issues/6695#issuecomment-2162560945), let's replace this with the already existing `ErrorHandler` component.

This may also warrant some small changes to the tests, e.g. no longer using the name `error boundary`.